packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/dropDownEditor.markup.tests.js

```diff
@@ -114,23 +113,6 @@ module('DropDownEditor markup', {
     });
 });
 
-
-module('aria accessibility', () => {
-    test('aria role', function(assert) {
```

Comment:
Seems like we missed this test for role now.

Reply:
No, this test already in the dropDownEditor.tests.js

Reply:
Moved all of ally tests into markup file: [0ff4d1d3afcafe181316848c94f2963d6e7e6cba].